internal/core/metadata/container/notifications.go

```diff
@@ -0,0 +1,14 @@
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/pkg/di"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"
+)
```

Comment:
Can these imports be sorted? I think this would match the rest of the code-base:
```go
import (
	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"

	"github.com/edgexfoundry/edgex-go/internal/pkg/di"
)

```

Reply:
FWIW, I sort edge-go before go-mod-core-contracts and put empty lines between each of the edgexfoundry modules' imports.

Reply:
Lets go with what @michaelestrin suggested. Ignore my suggestion.